### ISSUE TITLE
[ui-shell] Header and SideNav have elements not accessible via keyboard navigation

### ISSUE THREAD

**Issue by JordanWSmith15** [carbon-design-system/carbon-addons-iot-react#1838]
1. Open Storybook for [Header ](https://carbon-addons-iot-react.com/?path=/story/watson-iot-header--header-action-buttons-with-dropdowns )
or [Side-nav](https://carbon-addons-iot-react.com/?path=/story/watson-iot-sidenav--side-nav-component) ) in Firefox or Safari

Try to use the keyboard to navigate the elements in the screen

Actual Result: user cannot access the left side menu or top menu icons using the keyboard. 
On Safari, no focus states are shown at all. 
On Firefox things work better, but some items such as "Devices" in side-nav and "Profile" on header cannot be accessed. 

2. There are two viloations displayed in the Accessibility scan tab in storybook

Blocks https://jsw.ibm.com/browse/GRAPHITE-26091

**Comment by tay1orjones** [comment]
> On Safari, no focus states are shown at all.

It looks like safari by default ignores tabindex but has recently added a new accessibility option you have to enable:

![image](https://user-images.githubusercontent.com/3360588/100663129-78473c00-331b-11eb-885d-eab363a37193.png)

 Related https://github.com/carbon-design-system/carbon/issues/7257#issuecomment-724933452

**Comment by tay1orjones** [comment]
> On Firefox things work better, but some items such as "Devices" in side-nav and "Profile" on header cannot be accessed.

Yeah I can confirm the same - on Firefox these are not accessible via keyboard navigation and need to be looked into

**Comment by jkeen** [comment]
Checking into this on Firefox but keep running into #1950

**Comment by davidicus** [comment]
This is unfortunately default behavior for Mac OS. Both Safari and Firefox must turn on keyboard navigation for  links as outlined in this stack overflow. 
https://stackoverflow.com/questions/11704828/how-to-allow-keyboard-focus-of-links-in-firefox

I am not sure the wisdom of messing with tabbing behavior that goes against user settings. I would think most people using Firefox or Safari for a11y reasons would be aware of this and have their settings set appropriately. 

I am closing this issue for now. If you think there is more discussion to be had please reopen. 

Some of the bugs discussing this issue.
https://bugzilla.mozilla.org/show_bug.cgi?id=1036966
https://bugzilla.mozilla.org/show_bug.cgi?id=1436136
